spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -42,7 +41,7 @@
 	 * Whether meter IDs starting-with the specified name should be enabled. The longest
 	 * match wins, the key `all` can also be used to configure all meters.
 	 */
-	private Map<String, Boolean> enable = new LinkedHashMap<>();
+	private final Map<String, Boolean> enable = new LinkedHashMap<>();
```

Comment:
Can you please move the removal of Map setters to a separate PRs? I'd like to merge the rest in `2.0.x` but I can't merge this there as it's an API breaking change.